data-gathering/github_repos.json

```diff
@@ -63,7 +64,8 @@
 				{"name": "Opendigitalradio","orgs": ["Opendigitalradio"]},
 				{"name": "OpenOlitor","orgs": ["openolitor"]},
 				{"name": "School of Data in Switzerland","orgs": ["schoolofdata-ch"]},
-				{"name": "Open Source Challenge","orgs": ["opensource-challenge"]}
+				{"name": "Open Source Challenge","orgs": ["opensource-challenge"]},
+				{"name": "VeriSO","orgs": ["veriso"]}
```

Comment:
Actually VeriSO should also be added to Canton of Bern and Canton of Solothurn and Canton of Vaud plus the companies of the contributors. But for the moment we leave it like that.